rules/falco_rules.yaml

```diff
@@ -2428,6 +2437,12 @@
     and package_mgmt_procs
     and not package_mgmt_ancestor_procs
     and not user_known_package_manager_in_container
+  exceptions:
+   - name: kube-proxy
+     fields: [proc.cmdline, container.image.repository]
+     comps: [startswith, =]
+     values:
+       - [update-alternat, k8s.gcr.io/kube-proxy]
```

Comment:
We are not using _exceptions_ in the default ruleset (after a long discussion in the community that happened 1 year ago). I'd suggest rewriting that exclusion by using macros.

Reply:
@leogr This has been fixed now